packages/components/psammead-amp-geo/src/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eea: [
+      'at',
+      'be',
+      'bg',
+      'bl',
+      'cy',
+      'cz',
+      'de',
+      'dk',
+      'ee',
+      'es',
+      'fi',
+      'fr',
+      'gb',
+      'gf',
+      'gg',
+      'gi',
+      'gp',
+      'gr',
+      'hr',
+      'hu',
+      'ie',
+      'im',
```

Comment:
`im` isnt in the EEA

Reply:
Ahh ok apaz "...of countries in the EEA, also including Channel Islands and overseas territories."